magento/magento2
# Experius 2.3 patch catalog flat

### Description (*)

When the index type product flat is set to 'On Schedule' disabled products were not removed from the table until a full reindex.


### Manual testing scenarios (*)

1. Set the Flat Product to `On Schedule`
2. Disable the Test Product
3. Run the `indexer_update_all_views` cron with magerun2 or make sure the cron is running
4. As a result the product is still in the flat table
5. Apply changes
6. Enable the Test Product
7. Disable the Test Product
8. Run the `indexer_update_all_views` cron with magerun2 or make sure the cron is running
9. As a result the product is removed from the flat table

### Questions or comments


### Contribution checklist (*)
